pydatastructs/linear_data_structures/algorithms.py

```diff
@@ -1205,3 +1206,44 @@ def _prev_permutation_comp(x, y, _comp):
 
     return _permutation_util(array, start, end, comp,
                              _prev_permutation_comp)
+
+
+def bubble_sort(array,start = 0,):
```

Comment:
See the API of other sorting algorithms and modify this accordingly.

Reply:
Yeah Sure!!